src/components/hmi_message_handler/src/mb_controller.cc

```diff
@@ -235,6 +235,7 @@ void CMessageBrokerController::exitReceivingThread() {
   }
   acceptor_.close(ec);
   if (ec) {
+    // cppcheck-suppress unreadVariable
     std::string str_err = "ErrorMessage Close: " + ec.message();
```

Comment:
@Ypostolov I believe in this place is missed a line with logging, e.g. as in the line 234.